eng/Directory.Packages.props

```diff
@@ -293,13 +294,14 @@
     <PackageVersion Include="runtime.linux-x64.Microsoft.NETCore.ILAsm" Version="$(ILAsmPackageVersion)" />
     <PackageVersion Include="runtime.osx-x64.Microsoft.NETCore.ILAsm" Version="$(ILAsmPackageVersion)" />
     <PackageVersion Include="Basic.CompilerLog.Util" Version="0.6.1" />
-    <PackageVersion Include="Basic.Reference.Assemblies.NetStandard20" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net50" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net60" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net70" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net80" Version="1.4.5" />
-    <PackageVersion Include="Basic.Reference.Assemblies.Net461" Version="1.6.0" />
-    <PackageVersion Include="Basic.Reference.Assemblies.NetStandard13" Version="1.6.0" />
+    <PackageVersion Include="Basic.Reference.Assemblies.NetStandard20" Version="$(_BasicReferenceAssembliesVersion)" />
+    <PackageVersion Include="Basic.Reference.Assemblies.Net50" Version="$(_BasicReferenceAssembliesVersion)" />
+    <PackageVersion Include="Basic.Reference.Assemblies.Net60" Version="$(_BasicReferenceAssembliesVersion)" />
+    <PackageVersion Include="Basic.Reference.Assemblies.Net70" Version="$(_BasicReferenceAssembliesVersion)" />
+    <PackageVersion Include="Basic.Reference.Assemblies.Net80" Version="$(_BasicReferenceAssembliesVersion)" />
+    <PackageVersion Include="Basic.Reference.Assemblies.Net461" Version="$(_BasicReferenceAssembliesVersion)" />
+    <PackageVersion Include="Basic.Reference.Assemblies.NetStandard13" Version="$(_BasicReferenceAssembliesVersion)" />
+    <PackageVersion Include="Basic.Reference.Assemblies.Net90" Version="$(_BasicReferenceAssembliesVersion)" />
```

Comment:
><PackageVersion Include="Basic.Reference.Assemblies.Net90" Version="$(_BasicReferenceAssembliesVersion)" /> [](http://example.com/codeflow?start=4&length=107)

Did you intentionally dropped ``` <PackageReference Include="Basic.Reference.Assemblies.Net90" />``` from the test project file?

Reply:
Yes. In this PR I was trying to get the `Net90` reference out of the picture so I could validate my theory that the test baseline changes came from the update to `Net80` and were independent of `Net90`.